app/javascript/apiService.js

```diff
@@ -22,17 +22,10 @@ const getSupplementalApplication = async (applicationId) =>
       fileBaseUrl: file_base_url
     }))
 
-const getUnits = async (applicationId, listingId) =>
+const getUnits = async (listingId) =>
   request
-    .get(
-      `/supplementals/units`,
-      { params: { application_id: applicationId, listing_id: listingId } },
-      true
-    )
-    .then(({ available_units, units }) => ({
-      availableUnits: available_units,
-      units
-    }))
+    .get(`/supplementals/units`, { params: { listing_id: listingId } }, true)
+    .then((r) => r.units)
```

Comment:
We seem to have some cases where apiService returns `{thingName: thing}`, and some where we just return `thing`. I updated this one just return units straight because we're now only returning units and it seems pretty straightforward but @james-wills-sf let me know if you have a strong preference

Reply:
This is good to me. we can default to just returning the object itself unless there's more than one object in the response.